v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -76,42 +68,42 @@ public interface KafkaReadOptions extends PipelineOptions {
   void setConsumerGroupId(String value);
 
   @TemplateParameter.Enum(
-      order = 5,
+      order = 4,
       groupName = "Source",
       enumOptions = {
-        @TemplateParameter.TemplateEnumOption("earliest"),
-        @TemplateParameter.TemplateEnumOption("latest"),
+        @TemplateParameter.TemplateEnumOption(Offset.EARLIEST),
+        @TemplateParameter.TemplateEnumOption(Offset.LATEST),
       },
       optional = true,
       description = "Default Kafka Start Offset",
       helpText =
           "The Kafka offset to read from. If there are no committed offsets on Kafka, default offset will be used.")
-  @Default.String("latest")
+  @Default.String(Offset.LATEST)
   String getKafkaReadOffset();
 
   void setKafkaReadOffset(String value);
 
   @TemplateParameter.Enum(
-      order = 6,
+      order = 5,
       name = "kafkaReadAuthenticationMode",
       groupName = "Source",
       enumOptions = {
-        @TemplateParameter.TemplateEnumOption("NONE"),
-        @TemplateParameter.TemplateEnumOption("SASL_PLAIN"),
+        @TemplateParameter.TemplateEnumOption(KafkaAuthenticationMethod.SASL_MECHANISM),
```

Comment:
I see SASL_MECHANISM and SASL_PLAIN used in different places in this PR, but I think it's the same thing?
Should we remove SASL_MECHANISM and keep SASL_PLAIN everywhere?

Reply:
Based on the offline discussion
The enum would contain 
1. SASL_PLAIN
2. TLS
3. NONE

Is this right?